### ISSUE TITLE
Experiences with unit sphere normalization

### ISSUE THREAD

**Issue by TobiasMascetta** [HuguesTHOMAS/KPConv-PyTorch#208]
Hello Hugues,

From what I understand from your thesis and the code, an input normalization strategy (for example unit sphere normalization) is not necessary because of the grid-based input subsampling and the overall architecture. 
However, I am currently using a modified KPFCNN without reprojection and grid-based input subsampling for reconstruction and want to compare my results to other networks.

I was wondering whether you have any previous experiences with applying unit sphere normalization to the input of KPConv? 
I find it hard to tune the hyperparameters and the reconstruction results seem to get worse with input normalization.

Best regards
